informer/informer.go

```diff
@@ -98,6 +103,11 @@ func New(config Config) (*Informer, error) {
 	return newInformer, nil
 }
 
+func (i *Informer) Boot(ctx context.Context) error {
+	prometheus.MustRegister(prometheus.Collector(i))
```

Comment:
Again, we only should register in case it is not already registered. This will make the aws-operator and kvm-operator panic IMO.

Reply:
There isn't a way to check if a collector is already registered as far as I can find. 

I added `prometheus.Unregister(collector)` which should fix this issue. From the docs it should unregister the collector if it is registered and do nothing if it isn't registered.

Reply:
You can use `sync.Once`.